### ISSUE TITLE
Error running tests for isbn-verifier

### ISSUE THREAD

**Issue by fleimgruber** [exercism/julia#364]
When running the tests via

```
julia runtests.jl
```

we get the error

```
ERROR: LoadError: UndefVarError: ISBN not defined
Stacktrace:
 [1] top-level scope at /home/user/dev/exercism/julia/isbn-verifier/runtests.jl:5
 [2] include(::Function, ::Module, ::String) at ./Base.jl:380
 [3] include(::Module, ::String) at ./Base.jl:368
 [4] exec_options(::Base.JLOptions) at ./client.jl:296
 [5] _start() at ./client.jl:506
in expression starting at /home/user/dev/exercism/julia/isbn-verifier/runtests.jl:5
```

so this test file assumes that a user implemented the `ISBN` type, but that is not required from the description of the exercise.
I guess this comes from the expectation that a solution looks like https://github.com/exercism/julia/blob/c6b067b39fb5217c745fba23906248a19521c7be/exercises/practice/isbn-verifier/.meta/example.jl but a naive user (me) might just try to implement the `isvalid` function for a string (that's actually another issue, the exercise does not say that `isvalid` should take two arguments).

After adding

```
struct ISBN <: AbstractString
end
```

rerunning `julia runtests.jl` gives

```
Test Summary:      | Pass  Total
valid ISBN numbers |    4      4
Test Summary:        | Pass  Total
invalid ISBN numbers |   15     15
ERROR: LoadError: LoadError: UndefVarError: @isbn_str not defined
Stacktrace:
 [1] top-level scope
 [2] include(::Function, ::Module, ::String) at ./Base.jl:380
 [3] include(::Module, ::String) at ./Base.jl:368
 [4] exec_options(::Base.JLOptions) at ./client.jl:296
 [5] _start() at ./client.jl:506
in expression starting at /home/user/dev/exercism/julia/isbn-verifier/runtests.jl:53
in expression starting at /home/user/dev/exercism/julia/isbn-verifier/runtests.jl:51
```

So in summary I think the tests test more than the minimal requirements from the problem description.

**Comment by cmcaine** [comment]
This is part of a general issue with several exercises that the
instructions aren't a complete specification.

I think we should specify that users must define their own type.

**Comment by vntmb** [comment]
any update on this?
